client/asset/eth/doc.go

```diff
@@ -0,0 +1,13 @@
+//go:build gpl
+// +build gpl
+//
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+//
+// This package also imports go-ethereum code and so carries the burden of
+// go-ethereum's GNU Lesser General Public License.
+
+// Package eth implements methods to work with ethereum swap contracts and
+// transactions. The code is unique as it requires a more restrictive liscense
+// and connot be used in proprietary software.
+package eth
```

Comment:
As with all the files, `gofmt` is forcing a newline between the tags and other docs.  Let's follow: license, tags, package docs in separate blocks in that order:

```go
// This code is available on the terms of the project LICENSE.md file,
// also available online at https://blueoakcouncil.org/license/1.0.0.
//
// This package also imports go-ethereum code and so carries the burden of
// go-ethereum's GNU Lesser General Public License.

//go:build gpl
// +build gpl

// Package eth implements ...
package eth
```

Reply:
I think it's like this now.